### DISCUSSION TITLE
Please help me with "Please remove unsupported files to proceed"

### DISCUSSION THREAD

**Discussion by nullyoon** [iTwin/itwinjs-core#6474] [Q&A]
I dont know why this happened,some one can help me how to analysis this question???![image](https://github.com/iTwin/itwinjs-core/assets/67211031/c1a44e30-57af-456c-8bc7-00b44f14ccbb)

**Comment by snehaInmdr** [comment]
Can you please go to the gear icon on the top right corner of the screen and click on Save logs from the menu? Please share this log with us so we can see further details about this failure.

**Reply by nullyoon** [reply]
Sure,here is the logs,hope it is helpful
[iTwinSnapshot Diagnostics 2024-02-29T01-07-29.901Z.zip](https://github.com/iTwin/itwinjs-core/files/14441513/iTwinSnapshot.Diagnostics.2024-02-29T01-07-29.901Z.zip)

**Reply by nullyoon** [reply]
I'm waiting for the results of your analysis, I would appreciate it if you could solve the problem!
